texar/torch/modules/decoders/rnn_decoders.py

```diff
@@ -468,10 +468,9 @@ def default_hparams():
 
             `"type"`: str or class or instance
                 The attention type. Can be an attention class, its name or
-                module path, or a class instance. The class must be a subclass
-                of :class:`~texar.torch.core.AttentionMechanism`. If class name
-                is given, the class must be from modules :mod:`texar.torch.core`
-                or :mod:`texar.torch.custom`.
+                module path, or a class instance. If class name is given, the
+                class must be from modules :mod:`texar.torch.core` or
+                :mod:`texar.torch.custom`.
```

Comment:
"The class must be a subclass of :class:`~texar.torch.core.AttentionMechanism`." --- This sentence is deleted? Is it cuz the class does not need to be the subclass of `AttentionMechanism`? If it does need, then we shall explicit say it, and expose `AttentionMechanism` to users, and probably say that common attention classes are already implemented, such as `BahdanauAttention` (refer users to the list: https://texar-pytorch.readthedocs.io/en/latest/code/core.html#attention-mechanism)